.travis.yml

```diff
@@ -53,6 +53,27 @@ script:
         exercisename=$(basename "$exercise")
         pushd ${exercise}
 
+        examplename="stub"
```

Comment:
these lines have very high duplication with the stub-testing code below. I'm not sure if it's desirable that we deduplicate that.